### ISSUE TITLE
Docker images are not beeing updated 

### ISSUE THREAD

**Issue by andmagom** [taylorsilva/carvel-images#1]
Hello Taylor. 

It looks like all the images are not being updated since one year ago.

Could you take a look please ?

**Comment by taylorsilva** [comment]
yeah I had them updating on a Concourse I was running on an personal server in my home. Haven't gotten around to setting that up again.
